### ISSUE TITLE
cy_reg_led_on() should be used instead of cy_rgb_led_set_color()

### ISSUE THREAD

**Issue by noritan** [cypresssemiconductorco/CypressAcademy_MTB101_Files#1]
**Prerequisites**

Can you reproduce the problem reliably?

Did you check current release notes for known issues?

If this is not the latest release, have you checked newer releases? 

**Description**:

Got compile error for the function call cy_rgb_led_set_color()

On the page 61 of the document "MTB2-Ch-5a-PSoC-6-Core.pdf" a function cy_rgb_led_set_color() is used to change the RGB LED color.  But I got following compile error.

```
main.c:73:9: error: too many arguments to function 'cy_rgb_led_set_color'
         cy_rgb_led_set_color(CY_RGB_LED_COLOR_YELLOW, CY_RGB_LED_MAX_BRIGHTNESS);
         ^~~~~~~~~~~~~~~~~~~~
```

Because the function is declared as having one argument.  It is not applicable when the 2nd argument is removed because the LED brightness is not specified.

Instead of the function, cy_reg_led_on() function can be used which requires two arguments color and brightness.

**Steps to Reproduce:**

1. Follow the exercise

2.

3.

**Expected Result:**

No compile error

**Actual Result:**

Got compile error

**Frequency (1/1, 1/50, 1/100):**

1/1

**Build/Commit:**

Please specify the release, like 1.4.1. If there is no release, give us the commit for the code you used.
V1.4

**Target:**

The chip or kit you are working with
CY8CKIT-062-WiFi-BT

**Host OS and Version:**

Windows

**Compiler:**

Include the version!
Modus ToolBox 2.1

**Environment**

Provide the context surrounding this problem. Perhaps an IDE, a debug probe, an RTOS, etc. (and versions)
eclipse IDE

**Comment by rmbk19** [comment]
Hi @noritan , 

Yes, you are right. [cy_rgb_led_set_color()](https://cypresssemiconductorco.github.io/rgb-led/html/group__group__board__libs.html#ga4e0fea7dd904cfc4ed49707dbb0aae6d) takes in only one argument - uint32_t color. I have notified the repo owner regarding this and the fix will be out by next week. Thank you for pointing this out. 

The [RGB LED API Reference Manual](https://cypresssemiconductorco.github.io/rgb-led/html/index.html) has a similar code snippet and you can use that for reference.

Thanks and Regards, 
Rakshith

**Comment by rmbk19** [comment]
Fixed in version 1.4.1
